flake.nix

```diff
@@ -42,13 +42,23 @@
                 assertion = config.boot.loader.grub.device == "/dev/sda";
                 message = "garnix.server needs the boot.loader.grub.device to be \"/dev/sda\"";
               }
+              {
+                assertion = config.networking.useNetworkd == false;
+                message = "garnix.server needs networking.useNetworkd to be false";
+              }
+              {
+                assertion = config.networking.useDHCP;
+                message = "garnix.server needs networking.useDHCP to be true";
+              }
             ];
 
             fileSystems."/" = {
               device = "/dev/sda1";
               fsType = "ext4";
             };
             boot.loader.grub.device = "/dev/sda";
+            networking.useNetworkd = false;
+            networking.useDHCP = true;
```

Comment:
```suggestion
```

Since we have the assertions now, and these are the default values, I think they can go away now.

Reply:
Ah, good catch! Done in bccb6990.